src/components/ClientSection.tsx

```
@@ -0,0 +1,121 @@
+  terminerService,
+  nouveauClient,
+  passerJour
+}) => {
+  const firstActionRef = useRef<HTMLButtonElement>(null);
+
+  useEffect(() => {
+    if (clientActuel && firstActionRef.current) {
+      firstActionRef.current.focus();
+    }
+    // eslint-disable-next-line
+  }, [clientActuel && clientActuel.nom]);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)